### PR TITLE
Css two column layout - James Aguirre

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-04-02 at 12 38 57 PM

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
You seemed to have accidentally added `css-two-column-layout` to this as well. Please delete this folder like you did on the previous exercises, and then recommit / push.

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-whole">
+        <h2>SCALE, SPEED, AGILITY</h2>
+        <p>We've dedicated thousands of man hours to making healthcare institutins in the Helium network run 10x more efficiently and deliver an unparallel quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half"></div>
```
Comment (shawnkost) [inline css-two-column-layout/index.html:21]: You don't have any content inside this `column-half` `div`, you should have all of your content for this column inside here.

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <img src="images/software.svg" alt="software">
+      <div class="text">
+        <h4>INTUITIVE,<br>
+          SMART SOFTWARE</h4>
+        <p>We build products that transform how healthcare facilities run across Africa - making them radically more efficient
+          operationally and financially, a complete turnaround.</p>
+      </div>
+      <div class="column-half"></div>
```
Comment (shawnkost) [inline css-two-column-layout/index.html:29]: You don't have any content inside this `column-half` `div`, you should have all of your content for this column inside here.

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      <img src="images/practice.svg" alt="medicine">
+      <div class="text">
+        <h4>PRACTICE MEDICINE, NOT
+          <br> ADMINISTRATION</h4>
+        <p>Our all-in-one electronic health records (EHR), hospital management (HMIS) &amp; billing systems helps streamline your
+        facility, reduce waste and improve quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half"></div>
```
Comment (shawnkost) [inline css-two-column-layout/index.html:39]: You don't have any content inside this `column-half` `div`, you should have all of your content for this column inside here.

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      <img src="images/circumstance.svg" alt="for any circumstance">
+      <div class="text">
+        <h4>FOR ANY CIRCUMSTANCE</h4>
+        <p>Our intuitive system is built around the peculiarities of your facility; we support offline deployments, sync to the cloud and run on any device 24/7</p>
+      </div>
+      <div class="column-half"></div>
```
Comment (shawnkost) [inline css-two-column-layout/index.html:45]: You don't have any content inside this `column-half` `div`, you should have all of your content for this column inside here.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested James!

**Comment by james-aguirre** [comment]
Screenshot 2023-04-08 at 3 48 33 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**